asset/js/main.js

```diff
@@ -0,0 +1,68 @@
+const regex_check_special_characters =
+  /^[!@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?]*$/;
+
+var validRegexEmail =
```

Comment:
```suggestion
const regexCheckEmail =
```
Không sử dụng var nhé